fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -153,10 +154,6 @@ def __init__(
 
         # - setup buckets and tensor views
         model_size = sum([p.numel() for p in self.module.parameters()])
-        if dist.get_world_size(self.process_group) <= 8:
```

Comment:
this proved polemical (see https://github.com/facebookresearch/fairscale/issues/395, among others), and a lot of people at FB also did not understand why things behaved differently on one or more nodes, so I guess that it's a good signal